dds/DCPS/Service_Participant.h

```diff
@@ -308,6 +309,15 @@ class OpenDDS_Dcps_Export Service_Participant {
   }
 #endif
 
+#if defined(OPENDDS_NETWORK_CONFIG_UPDATER)
```

Comment:
Do we need these to be available on the ServiceParticipant API? I'm not sure why these methods exist.